Gemfile.lock

```diff
@@ -245,33 +247,34 @@ GEM
     method_source (1.0.0)
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.1009)
-    mimemagic (0.3.4)
+    mime-types-data (3.2020.0512)
+    mimemagic (0.3.5)
     mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
     minitest (5.14.1)
     msgpack (1.3.3)
     mysql2 (0.5.3)
-    net-scp (2.0.0)
-      net-ssh (>= 2.6.5, < 6.0.0)
-    net-ssh (5.2.0)
+    net-scp (3.0.0)
+      net-ssh (>= 2.6.5, < 7.0.0)
+    net-ssh (6.0.2)
     netrc (0.11.0)
     nio4r (2.5.2)
     nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
-    oj (3.10.5)
+    oj (3.10.6)
```

Comment:
### [oj](http://www.ohler.com/oj/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.10.5 → 3.10.6

[_[change-log](https://github.com/ohler55/oj/blob/master/CHANGELOG.md), [source-code](https://github.com/ohler55/oj)_]





<details>
<summary>Commits</summary>

A change of **2** commits. See the full changes on [the compare page](https://github.com/ohler55/oj/compare/v3.10.5...v3.10.6).

These are the individual commits:
- (d223a50) [fix API doc (#589)](https://github.com/ohler55/oj/commit/d223a50ea9cd19261b7b04caeab8e81c610fe662)
- (0c31f9d) [Skip of _ attribute continues instead of returning](https://github.com/ohler55/oj/commit/0c31f9da75c72bc7250573d8bc16dddbf2dce5f2)


</details>